proto/ligato/vpp/wireguard/wireguard.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package ligato.vpp.wireguard;
+
+option go_package = "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/wireguard;vpp_wg";
+
+message Peer {
+  // Public-key base64
+  string public_key = 1;
+
+  // Peer UDP port
+  uint32 port = 2;
+
+  // Keepalive interval (sec)
+  uint32 persistent_keepalive = 3;
+
+  //  Table ID
+  uint32 table_id = 4;
+
+  // Endpoint IP
+  string endpoint = 5;
+
+  // Wireguard interface name
+  string interface = 6;
```

Comment:
I'm not sure here.. but the name `Interface` might be potentially confusing to users. At first I thought it defines the interface name of the peer.. but I suspect it's the name of the actual interface created by wireguard, right?

[In similar cases](
https://github.com/ligato/vpp-agent/blob/master/proto/ligato/vpp/interfaces/interface.proto#L228) we use `*_if_name` suffix, perhaps `wireguard_if_name` or `wg_if_name` might make it more different to other `interface` / `name` fields that refer to the logical name of interfaces used in the vpp-agent internally, even though it's more "ugly". Perhaps just provide more info in the field comment to make it clear to users what it refers to.

CC @milanlenco

Reply:
You are right.
Done